src/main/resources/application.yaml

```diff
@@ -31,6 +33,10 @@ spring:
     testOnBorrow: true
     validationQuery: SELECT 1
 
+flyway:
+  schemas:
+    - ${JOB_SCHEDULER_DB_NAME:jobscheduler}
```

Comment:
read that there is no need to define schema and just use default `public`. otherwise, i wonder if spring data source sees the tables migration executed into